src/metpy/plots/declarative.py

```diff
@@ -1383,6 +1385,20 @@ class PlotObs(HasTraits):
     reduce_points = Float(default_value=0)
     reduce_points.__doc__ = """Float to reduce number of points plotted. (optional)"""
 
+    plot_units = List(default_value=[None], allow_none=True)
+    plot_units.__doc__ = """A list of the desired units to plot the fields in.
+
+    Setting this attribute will convert the units of the field variable to the given units for
+    plotting using the MetPy Units module, provided that units are attached to the DateFrame.
+    """
+
+    vector_plot_units = Unicode(default_value=None, allow_none=True)
+    vector_plot_units.__doc__ = """The desired units to plot the vector field in.
+
+    Setting this attribute will convert the units of the field variable to the given units for
+    plotting using the MetPy Units module, provided that units are attached to the DateFrame.
+    """
+
```

Comment:
Tiny docstring spelling error? "**Date**Frame" in 1392/1399

Reply:
Wow. Nice one Kevin...got it.

Reply:
Got one more in the copied string below, line 1399!

Reply:
GDMFH. Got it.

Reply:
Hahaha, perfect! Thanks Kevin